packages/react-native-platform-override/src/e2etest/collateral/0.62.2/flowconfig.windows.conflict

```diff
@@ -87,6 +87,8 @@ untyped-type-import
 [version]
 <<<<<<< Upstream
 ^0.113.0
+||||||| base
```

Comment:
I'm not sure if our tests are affected by local state on my machine or if @types files can have an influence on the diff we generate. I ran into this change when bumping versons for the appium replacement too.

Reply:
And it doesn't reply on the mac... Will investigate what state of my devbox is leaking into the tests....

Reply:
hmm, it just uses git diff so it relies on local git config :(

Reply:
too many options to git diff and can't find anything in `git config --list` that might cause this... I'll just accept my tests don'r pass locally...